immuni_analytics/models/swagger.py

```diff
@@ -0,0 +1,60 @@
+#   Copyright (C) 2020 Presidenza del Consiglio dei Ministri.
+#   Please refer to the AUTHORS file for more information.
+#   This program is free software: you can redistribute it and/or modify
+#   it under the terms of the GNU Affero General Public License as
+#   published by the Free Software Foundation, either version 3 of the
+#   License, or (at your option) any later version.
+#   This program is distributed in the hope that it will be useful,
+#   but WITHOUT ANY WARRANTY; without even the implied warranty of
+#   MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+#   GNU Affero General Public License for more details.
+#   You should have received a copy of the GNU Affero General Public License
+#   along with this program. If not, see <https://www.gnu.org/licenses/>.
+
+from sanic_openapi import doc
+
+
+class AuthorizationBody:
+    """
+    Doc model for the token authorization request body.
+    """
+
+    analytics_token = doc.String(
+        "The analytics_token to authorize for the operational_info uploads.",
+    )
+    device_token = doc.String("The device token to check against Apple DeviceCheck.")
+
+
+class AppleOperationalInfo:
+    """
+    Doc model for the operational info upload request body.
+    """
+
+    province = doc.String("The user's province of residence.")
```

Comment:
minor: OCD, please order them alphabetically

Reply:
We'll probably need to align with the official documentation on this.